cms/templates/js/course-outline.underscore

```diff
@@ -142,8 +142,19 @@ if (xblockInfo.get('graded')) {
                     </p>
                 </div>
             <% } %>
+            <% if (xblockInfo.get('is_time_limited')) { %>
+                <div class="status-timed-proctored-exam">
+                    <p>
+                        <span class="sr status-proctored-exam-label"> <%= gettext('Proctored Exam:') %> </span>
```

Comment:
Should this be parallel with line 150? This ends with a colon and the other does not. Recommend not using colons in UI.